src/.env

```diff
@@ -54,3 +47,7 @@ SENTRY_URL=''
 TOP_GG_TOKEN=''
 TWITCH_CLIENT_ID=''
 TWITCH_TOKEN=''
+
+# The database URL for Prisma, needs to be copied in a local `.env` at the root
+# for Prisma's operations to work.
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/skyra?schema=public"
```

Comment:
Why dont we just put .env in the root altogether and then we can simplify setup from skyra env-utilities as well because it will default to the root.

And for prod we can inline the envs in docker-compose just like we do for the other bots

Reply:
Because then you won't be able to override it with `.env.development.local`, Prisma doesn't do the dotenv-cra behaviour afaik

Reply:
You're right